src/main/java/com/pscheduler/server/model/GpaCourse.java

```diff
@@ -0,0 +1,42 @@
+package com.pscheduler.server.model;
+
+import javax.persistence.*;
```

Comment:
No need for the * import since we don't have that many from the persistence lib

Reply:
Done